src/App.js

```diff
@@ -2,13 +2,16 @@ import React from 'react';
 import { BrowserRouter as Router, Route } from 'react-router-dom';
 
 import { Home } from './pages';
+import { Details } from './pages';
```

Comment:
Se o import é do mesmo local, chama aninhado
```suggestion
import { Home, Details } from './pages';
```